### DISCUSSION TITLE
模糊拼音支持

### DISCUSSION THREAD

**Discussion by imbamboo** [KyleBing/rime-wubi86-jidian#92] [Ideas]
我平时五笔+拼音混合输入。
用的wubi86_jidian_pinyin.schema.yaml 这个配置。
但拼音好象不支持模糊拼音，有时拼不出来。
有没有支持**模糊拼音**的配置文件？谢谢。

**Comment by KyleBing** [comment]
模糊拼音    是指 L N  z zh c ch s sh 这种吗？

**Reply by imbamboo** [reply]
感谢回复，是的，正是你说的那种。

**Reply by imbamboo** [reply]
找到民间配方表了了
https://www.cnblogs.com/F4NNIU/p/16994561.html

**Reply by KyleBing** [reply]
> 找到民间配方表了了 https://www.cnblogs.com/F4NNIU/p/16994561.html

嗯，可以
